docs/content-flow.md

```diff
@@ -0,0 +1,388 @@
+# Content Flow
+
+A major goal of containerd is to create a system wherein content can be used for executing containers.
+In order to execute on that flow, containerd requires content and to manage it.
+
+This document describes how content flows into containerd, how it is managed, and where it exists
+at each stage in the process. We use an example of going from a known image
+[docker.io/library/redis:5.0.9](https://hub.docker.com/layers/redis/library/redis/5.0.9/images/sha256-4ff8940144391ecd5e1632d0c427d95f4a8d2bb4a72b7e3898733352350d9ab3?context=explore) to explore the
+flow of content.
+
+## Content Areas
+
+Content exists in several areas in the containerd lifecycle:
+
+* OCI registry, for example [hub.docker.com](https://hub.docker.com) or [quay.io](https://quay.io)
+* containerd content store, under containerd's local storage space, for example, on a standard Linux installation at `/var/lib/containerd/io.containerd.content.v1.content`
+* snapshots, under containerd's local storage space, for example, on a standard Linux installation at `/var/lib/containerd/io.containerd.snapshotter.v1.<type>`. For an overlayfs snapshotter, that would be at `/var/lib/containerd/io.containerd.snapshotter.v1.native`
```

Comment:
not native

Reply:
Good point. The example even lists it in the right place, but I made the mistake when typing it in. Fixing.